app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -67,7 +86,6 @@ def handle_valid_otp_delivery_preference(method)
       end
 
       send_user_otp(method)
-      session[:code_sent] = 'true'
```

Comment:
I confirmed this isn't used in the code anywhere so it's removal is ok.  Were there plans to use it somewhere?

Reply:
Good question. This was first introduced in https://github.com/18F/identity-idp/pull/572, but it looks like we're not displaying that flash anymore, which to me seems like a bug. I'll have to look into it.